superset/utils/core.py

```diff
@@ -1064,8 +1064,8 @@ def get_since_until(
             since, until = time_range.split(separator, 1)
             if since and since not in common_time_frames:
                 since = add_ago_to_since(since)
-            since = parse_human_datetime(since)
-            until = parse_human_datetime(until)
+            since = parse_human_datetime(since)  # type: ignore
```

Comment:
Overriding of variable names of different types should be avoided in the future.

Reply:
Agree, we could use a different variable here